### ISSUE TITLE
Investigate a crashing converted game

### ISSUE THREAD

**Issue by Idhrendur** [Idhrendur/paradoxGameConverters#224]
See here:
https://forum.paradoxplaza.com/forum/index.php?threads/the-eu4-to-v2-converter-development-thread.707362/page-221#post-22413517

**Comment by derHochmeister** [comment]
What does this say? (Can't access forums, if I know problem, I may find solution)

**Comment by Idhrendur** [comment]
"Hey there,

First of all, thanks for all the effort that's gone into making the converter. I could really use some help though! I converted a game using the latest release (1.0E) and for some reason it keeps crashing every time I reach May 26, 1845. No idea why this is happening. This is part of a megacampaign I've been playing from CK2 from the 869 start date, so help would really, really be appreciated! I've attached the mod file and linked to the save, I'm also using the CORES 1.1 mod as well."

The save is attached to the post.
